src/components/dialogs/exit.vue

```diff
@@ -22,18 +22,22 @@
 
 <script lang="ts">
 import { Component } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
 import { DisplayedComponentMixin } from '@/mixins'
-import newReqModule from '@/store/new-request-module'
+import { ActionBindingIF } from '@/interfaces/store-interfaces'
 
 @Component({})
 export default class ExitDialog extends DisplayedComponentMixin {
+  @Getter getExitModalVisible!: boolean
+  @Action setExitModalVisible!: ActionBindingIF
+
   get showModal () {
-    return newReqModule.exitModalVisible
+    return this.getExitModalVisible
   }
   set showModal (value: boolean) {
   }
   hideModal () {
-    newReqModule.mutateExitModalVisible(false)
+    this.setExitModalVisible(false)
```

Comment:
Have you formed an idea how to get rid of this visibility state? It feels wrong but I don't have a solution yet. Maybe we can treat like our other dialogs: the parent passes in the visibility prop? I dunno, is that any better? I don't mind our implementation.